spec/services/search/vacancy_search_spec.rb

```diff
@@ -171,6 +171,7 @@
         before do
           allow(vacancies).to receive(:empty?).and_return(false)
           mock_algolia_search(vacancies, 1, keyword, arguments_to_algolia)
+          freeze_time
```

Comment:
I don't understand what the freeze_time is for - what depends on time?

Reply:
The timestamp in the filters